src/python/pants/notes/2.17.x.md

```diff
@@ -1,5 +1,54 @@
 # 2.17.x Release Series
 
+## What's New
+
+### Overall
+
+The [`scie-pants` launcher binary](https://github.com/pantsbuild/scie-pants) is now the recommended way to run Pants.
+Use of a `./pants` shell script in a repository is deprecated. Among other benefits, the `scie-pants` launcher is able
+to automatically download and maintain a Python distribution for running Pants. It also allows you to run `pants`
+from any directory within a repository and not just the build root. See the [Pants  installation
+instructions](https://www.pantsbuild.org/docs/installation) for additional information on how to migrate to `scie-pants.`
+
+### Backends
+
+#### Python
+
+Python dependencies are now extracted via an engine intrinsic for increased performance.
+
+When exporting `python_distribution` targets, they can be installed as editable installs in the exports.
+
+#### Shell
+
+The behavior of the `workdir` field on several shell-backend target types including `shell_command` has been
+updated to refer to the repository's build root when an empty string is set.
+
+For the `adhoc_tool` target type, the `stdout` and `stderr` fields are now specified relative to the applicable
+workdir and also support absolute paths.
+
+#### New: Javascript
```

Comment:
@tobni: Is the Javascript ready for mention in the release notes?

Are there other contributors who should be mentioned for this support?

Reply:
Maybe a call for testing and/or disclaimer that it is in early stages? Theres some showstopper bugs in there which might frustrate...

Otherwise I believe this release contains test goal support and enables choice of package managers and a push for docker backend compatability (Ill have to double check though.)

I think 2.17  contains my contributions only, but @theoribeiro has helped me course correct.

Reply:
@tobni : It would probably be good to create a "tracking ticket" that covers the issues you would like to see resolved before it is no longer experimental. It's a young convention (not actually documented anywhere yet), but useful for linking from various places.

Reply:
> Maybe a call for testing and/or disclaimer that it is in early stages? Theres some showstopper bugs in there which might frustrate...

> Otherwise I believe this release contains test goal support and enables choice of package managers and a push for docker backend compatability (Ill have to double check though.)

Any preferred wording for these points?

Reply:
Here's the tracking ticket: https://github.com/pantsbuild/pants/issues/19240, I'll see if I can come up with a list of changes.